plugin.js

```diff
@@ -78,11 +78,21 @@ export default async (ctx, inject) => {
 
   // Firebase Analytics can only be initiated on the client side
   if(process.browser && options.useOnly.includes('analytics')) {
-    <%= options.useOnly.includes('analytics') ? "await import('firebase/analytics')" : "" %>
+    await import('firebase/analytics')
 
     const fireAnalytics = firebase.analytics()
     const fireAnalyticsObj = firebase.analytics
     inject('fireAnalytics', fireAnalytics)
     inject('fireAnalyticsObj', fireAnalyticsObj)
   }
+
+  // Firebase Remote Config can only be initiated on the client side
+  if(process.browser && options.useOnly.includes('remoteConfig')) {
+    await import('firebase/remote-config')
+
+    const fireConfig = firebase.remoteConfig()
+    const fireConfigObj = firebase.remoteConfig
+    inject('fireConfig', fireConfig)
```

Comment:
What about we add the following functionality here?

```js
const optionalFetchInterval = options.remoteConfig.minimumFetchIntervalMillis
remoteConfig.settings = {
  minimumFetchIntervalMillis: optionalFetchInterval ? optionalFetchInterval : 3600000,
};

remoteConfig.defaultConfig = (options.remoteConfig.defaultConfig)
```

So remoteConfig is setup completely and the default values can, if needed, be passed through nuxt.config.js.

Reply:
That was just a quick write up, need to do it better since this will break if options.remoteConfig doesn't exist.